textBasedAdventure.js

```diff
@@ -1,5 +1,246 @@
 const readline1 = require('readline-sync')
 
-let nameInput = readline.question("Enter your name: ")
+let nameInput = readline1.question("Enter your name: ")
+    console.log(`I'm pleased to meet you ${nameInput}!`)
+    let askOen =  readline1.keyIn("Im ready to go around the world Lalalala, are you set?")
+    console.log(` Nice! can't wait to show you cool fun Fact we learnt in each country`)
+    console.log("Each time we arrive at our destination we will need to Unscramble the word ");
+    console.log("you will have up to three tries to get it RIGHT");
 
-console.log(`Hello ${nameInput}!  Welcome to my game.`)
+function tryInGame () {
+    let question = readline1.keyInYN(`Do you want to fly around the world Lalalala`)
+    if( question === true) {
```

Comment:
can move readline1.keyInYN(`Do you want to fly around the world Lalalala`) as the conditional